website/docs/r/linux_virtual_machine.html.markdown

```diff
@@ -248,6 +248,8 @@ A `os_disk` block supports the following:
 
 -> **NOTE:** The Disk Encryption Set must have the `Reader` Role Assignment scoped on the Key Vault - in addition to an Access Policy to the Key Vault
 
+-> **NOTE:** Encryption type of the key will be decided by the disk encryption set. [More info on encryption type](https://docs.microsoft.com/en-us/azure/virtual-machines/disk-encryption)
```

Comment:
this fields configured on the Disk Encryption Set rather than the VM, so this comment isn't really necessary:

```suggestion
```

Reply:
Removed.